### ISSUE TITLE
Wrong class in documentation for function getFileContentFromWorker

### ISSUE THREAD

**Issue by YngveNPettersen** [buildbot/buildbot#4085]
The documentation page <http://docs.buildbot.net/1.1.1/developer/cls-buildsteps.html#commandmixin> says that the function getFileContentFromWorker is part of the class buildbot.process.buildstep.CommandMixin .

As far as I can tell the function is only found in buildbot.steps.worker.CompositeStepMixin , and there is no class inheritance between the two classes. Additionally, it seems the other functions listed in the section are duplicated in that class.

**Comment by tardyp** [comment]
Indeed, the two mixin classes are largely overlapping each other. Would you like to make a PR to only keep buildbot.process.buildstep.CommandMixin, and to alias buildbot.steps.worker.CompositeStepMixin  to the other one?

**Comment by YngveNPettersen** [comment]
I don't think so, sorry; at the moment I am just feeling my way around the source and API trying to figure out how to get my system ported up (read:. rewritten from scratch) from the heavily modified 0.8.4p1 we have been using.

**Comment by tardyp** [comment]
I sympathise with you :)
I can only argue for you not to do the same mistakes and contribute your changes.

**Comment by YngveNPettersen** [comment]
In this case I am planning to stay entirely away from locally patching the buildbot code (the old system is based on the chromium buildbot. I do have to write some subclasses to add things I need, but have kept that to a minimum so far.
